lib/avatax/api.rb

```diff
@@ -1,5 +1,6 @@
 require File.expand_path('../connection', __FILE__)
 require File.expand_path('../request', __FILE__)
+require 'active_support'
```

Comment:
Why is active_support required in api.rb? @orien

Reply:
See #143

Reply:
I've removed this and the build still works.